elasticsearch/elasticsearch.go

```diff
@@ -63,7 +66,7 @@ func (b *BulkIndexer) Add(r elastic.BulkableRequest) error {
 }
 
 func (b *BulkIndexer) Flush() error {
-	_, err := b.bulkService.Do()
+	_, err := b.bulkService.Do(context.TODO())
```

Comment:
Yeah, even if I was annoyed having to add them all, it'd be cool to be able to add in stuff like cancellation, deadlines, etc. if we want them.